Lib/test/test_typing.py

```diff
@@ -473,7 +473,6 @@ def test_var_substitution(self):
 
     def test_bad_var_substitution(self):
         T = TypeVar('T')
-        P = ParamSpec("P")
```

Comment:
`ParamSpec.__typing_subst__` is tested in another place in this file:

https://github.com/python/cpython/blob/7894bbe94ba319eb650f383cb5196424c77b2cfd/Lib/test/test_typing.py#L7528-L7539